lib/src/download_utils.dart

```diff
@@ -8,12 +8,12 @@ import 'dart:io';
 
 import 'package:http/http.dart' as http;
 import 'package:http/retry.dart' as http_retry;
+import 'package:pana/pana.dart';
```

Comment:
Let's use direct import of the generated version instead.

Reply:
++ 

Thanks - I messed this up